addons/core/translations/actions/en-us.yaml

```diff
@@ -38,6 +38,9 @@ add-hosts: Add Hosts
 set-password: Set Password
 change-password: Change Password
 edit-form: Edit Form
+edit-user-details:
+  button-text: Edit User details
+  aria-label: 'edit form: users'
```

Comment:
We typically keep generic actions in this file. With this being unique to User, I would recommend adding this to the user.actions translation found [here](https://github.com/hashicorp/boundary-ui/blob/main/addons/core/translations/resources/en-us.yaml#L570).